# motion def:
motion_file: allan_variance_analysis.csv
# IMU params:
imu: 1
# sample frequency of simulated GNSS/IMU data:
sample_frequency:
    imu: 100.0
    gps: 10.0
# topic name:
imu_topic_name: /sim/sensor/imu
odom_gt_topic_name: /pose/ground_truth
# output rosbag path:
output_path: /workspace/data/gnss_ins_sim/allan_variance_analysis
# output name:
output_name: data.bag
groundtruth_output_path: /workspace/data
groundtruth_output_name: odom_groundtruth.txt